02-express-tutorial/authorize.js

```diff
@@ -0,0 +1,18 @@
+const authorize = (req, res, next) => {
+    
+    // const { user } = req.query;
+    // if (user === 'john') {
+    //   req.user = { name: 'john', id: 3 };
+    // //if this condition is met we go to next()-->
+
+    //   next();
+    // } else {
+    //   res.status(401).send('Unauthorized');
+    // }
+    // it will work id we provide data: user:'john in url: http://localhost:5000/?user=john
+    //otherwise there will be an error 401 and 'Unauthorized'
```

Comment:
Just checking in; did these commented out parts work for you and you just commented them out for easier development during the later tasks; or did something break?

Reply:
exactly, I just wanted to save this part of code for the future tasks